programmers/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,4 @@
+function solution(s) {
+  var answer = (s.length % 2) ? s[Math.floor(s.length / 2)] : s.substr(Math.floor(s.length / 2) - 1, 2)
```

Comment:
혹시 Prettier 돌리셨나요?ㅋㅋㅋㅋ 왤케 코드가 이쁘지 굳굳!!
근데 실제 개발할 때는 한줄코드보다 명확하게 분리해주는게 더 좋은 코드랍니다 :D
```suggestion
  const answer = (s.length % 2) ? s[Math.floor(s.length / 2)] : s.substr(Math.floor(s.length / 2) - 1, 2)
```